spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/async/LazyTraceExecutor.java

```diff
@@ -25,12 +25,14 @@
 import org.springframework.beans.factory.NoSuchBeanDefinitionException;
 import org.springframework.cloud.sleuth.DefaultSpanNamer;
 import org.springframework.cloud.sleuth.SpanNamer;
-import org.springframework.cloud.sleuth.Tracer;
 import org.springframework.cloud.sleuth.TraceKeys;
+import org.springframework.cloud.sleuth.Tracer;
 
 /**
- * @author Dave Syer
+ * {@link Executor} that wraps {@link Runnable} in a {@link LocalComponentTraceRunnable}.
```

Comment:
I doubt LocalComponentTraceRunnable is a very public type. Maybe explain what this executor does (by wrapping with LocalComponentTraceRunnable)?